app/api.py

```diff
@@ -46,12 +46,37 @@ def fetch_from_api():
             logging.critical(err.status_code + 'Response from API..')
         
         return
+    database_data = db.session.query(Car).all()
+    list_sync = []
+    for j in data['results']:
+        try:
+            list_sync.index(j['objectId'])
+        except:
+            list_sync.append(j['objectId'])
 
+    for i in database_data:
+        try:
+            list_sync.index(i.id)
+        except:
+            deleting = db.session.query(Car).filter_by(id = id.i).delete()
+            db.session.commit()
     for i in data['results']:
         check = db.session.query(Car).filter_by(id=i['objectId']).first()
         if not check:
             car = Car(id=i['objectId'], year=i['Year'], make=i['Make'], created_at=i['createdAt'],
                       updated_at=i['updatedAt'])
             db.session.add(car)
             db.session.commit()
-        
+        else:
+            if check.updated_at == i['updatedAt']:
+                pass

```

Comment:
can these if conditions be made better?

Reply:
sure, I will check and try to implement another approach to this problem if possible.